composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 | ^7.4 | ^8.0",
```

Comment:
```suggestion
        "php": "^7.3 || ^8.0",
```

Reply:
This does the same thing.....just comes down to personal preference. I always like clearly defining each version. If you dont then ill change it.